src/io/github/sspanak/tt9/ime/helpers/TextField.java

```diff
@@ -335,7 +335,7 @@ public int getAction() {
 			return field.actionId;
 		}
 
-		int standardAction = field.imeOptions & (EditorInfo.IME_MASK_ACTION | EditorInfo.IME_FLAG_NO_ENTER_ACTION);
+		int standardAction = field.imeOptions & (EditorInfo.IME_MASK_ACTION | EditorInfo.IME_FLAG_NO_ENTER_ACTION | EditorInfo.IME_FLAG_NAVIGATE_NEXT);
```

Comment:
This is not the appropriate place for changing the action. Please follow check [my previous comment](https://github.com/sspanak/tt9/pull/365#issuecomment-1698595085) for more details